src/components/DisplayBox.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
```

Comment:
Doesn't look like this file is used anywhere? If not you can go ahead and remove it.

Reply:
Removed it